email_template_qweb/models/mail_template.py

```diff
@@ -26,7 +26,9 @@ def generate_email(self, res_ids, fields):
         ).items():
             self_with_lang = self.with_context(lang=lang)
             for res_id in res_ids:
-                if self.body_type == "qweb" and (not fields or "body_html" in fields):
+                if self.body_type == "qweb_view" and (
+                    not fields or "body_html" in fields
+                ):
```

Comment:
can't you by now override https://github.com/odoo/odoo/blob/15.0/addons/mail/models/mail_render_mixin.py#L565 and just pass the value of `body_type` in `engine`?

Reply:
That actually renders a mail template, while this tries to take the predefined [views](https://github.com/OCA/social/pull/1072/files#diff-3111cb97ab1476860a01e335e26ed4196c9535d5dae0bce9a9358c4668e28ab3R15) to render the mail's body

Reply:
I know what the module is doing because I'm it's original author ;-)

But now I see this functions insists on views with an [xmlid](https://github.com/odoo/odoo/blob/15.0/addons/mail/models/mail_render_mixin.py#L378) for that case, so it's indeed not usable for our case